README.md

```diff
@@ -190,7 +190,7 @@ require 'frozen_record/test_helper'
 
 class CountryTest < ActiveSupport::TestCase
   setup do
-    test_fixtures_base_path = Rails.root.join(%w(test support fixtures))
+    test_fixtures_base_path = Rails.root.join(*%w(test support fixtures))
```

Comment:
`Rails.root.join('test/support/fixtures'')`

Reply:
True, this also works. Was wondering which would be preferred. Will update.